prep-html-skeleton/index.html

```diff
@@ -0,0 +1,7 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Skeleton Exercise</title>
+    <body>
```

Comment:
Hi Avalon, be sure to close your `<head>` tag before opening your `<body>` tag.

Reply:
![image](https://user-images.githubusercontent.com/96964291/149066088-09a88ffd-e1e2-42df-a2cb-c8466ba7fb49.png)

![image](https://user-images.githubusercontent.com/96964291/149066147-cc29f690-0a03-425c-9a9d-df56663c808e.png)

I believe it is properly corrected now, thank you for the feedback!